rpa-builder/6.2/modules/ROOT/pages/toolbox-network-rest-call.adoc

```diff
@@ -1,170 +1,247 @@
 
-
 = REST Call
 
-You can use the _REST Call Action Step_ to fetch data directly from a
+Use the REST Call Action Step to fetch data directly from a
 server and to collect the required information in a simple way.
 
-You define the end point as you wish and you can assign parameters or
-process variables to the call. This enables you to flexibly address any
-REST interface. The response in JSON format (or XML) can be used as a
+You define the endpoint according to your needs and you can assign parameters or

```

Comment:
Some day it will be good to remove the "You..." from some of the text and have the text be more direct: "Define the endpoint according to your needs and assign..."